.github/workflows/sync.yml

```diff
@@ -0,0 +1,92 @@
+name: Sync
+
+on:
+  workflow_dispatch:
+
+jobs:
+  sync:
+    strategy:
+      matrix:
+        go-version: [1.19.x]
```

Comment:
I don't think you need this Go version, but there does need to be a Python version & setup step (unless that's baked in somewhere else?).